_posts/0000-01-02-knandini741.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
this is looking good. i felt a few comments that should be addressed before this gets merged.